src/main/resources/lang/english.lang

```diff
@@ -1297,6 +1297,13 @@ spawn reasons:
 	trap: trap
 	village_defense: village defense, golem defense, iron golem defense
 	village_invasion: village invasion, village invading
+	
+# -- Difficulties --
+difficulty:

```

Comment:
```suggestion
difficulties:
```

Reply:
The name of the Spigot enum is `difficulty` this is what is used to allocate it towards it's types.

Reply:
It doesn't work in this way you know, "spawn reason**s**" is not a class name.